src/main/java/com/hydratereminder/HydrateReminderPlugin.java

```diff
@@ -299,11 +298,27 @@ private void handleHydratePrevCommand()
 		final int hours = Math.toIntExact(timeSinceLastBreak.toHours());
 		final int minutes = Math.toIntExact(timeSinceLastBreak.toMinutes() % 60);
 		final int seconds = Math.toIntExact((timeSinceLastBreak.toMillis() / 1000) % 60);
-		final String timeString = String.format("%s hours %s minutes %s seconds since the last hydrate break",
-				hours, minutes, seconds);
+		final String timeString = timeDisplay(hours, minutes, seconds);
 		sendHydrateEmojiChatMessage(ChatMessageType.GAMEMESSAGE, timeString);
 	}
 
+	/**
+	 * <p>Handle the String formatting of the specified time.</p>
+	 * @param hours
+	 * @param minutes
+	 * @param seconds
+	 * @return Time in a string format
+	 * @since 1.1.1
+	 */
+	String timeDisplay(int hours, int minutes, int seconds) {
```

Comment:
Can you also take a look into fixing this code quality alert?
Looks like it expects explicit scoping to be done here.

https://app.codacy.com/gh/jmakhack/hydrate-reminder/pullRequest?prid=7761949

Reply:
Ok i'll make it public.